### PR TITLE
Update App.jsx

### PR DESCRIPTION
**Pull request by sumiczek**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
Create a counter component that shows the counted amount, and it is possible to increment and decrement this value.

### Requirements
1. Counter component should be implemented as a reusable React component.
2. The component should display the counted amount and provide functionality to increment and decrement these values.
3. Counted amount cannot be below 0.

### Code Changes Review

The provided code changes in `src/App.jsx` file do not cover the task requirements. The changes only modify the text displayed by the App component from "Lights generator" to "Lights generatorrrr". 

There is no implementation of a counter component, nor any functionality to increment or decrement a value. Also, there is no logic to ensure that the counted amount cannot be below 0. 

In conclusion, the code changes do not meet any of the task requirements.
